backend/app/Sync/Shapes/PalletMovementsRecentShape.php

```
@@ -0,0 +1,54 @@
+            'worker_id',
+            'from_location',
+            'to_location',
+            'moved_at',
+            'notes',
+            'performed_by',
+            'created_at',
+            'updated_at',
+        ];
+    }
+
+    public function where(User $user): ?string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)